backend/shared/rest/src/main/java/com/meemas/shared/BooleanExpression.java

```diff
@@ -0,0 +1,16 @@
+package com.meemas.shared;
```

Comment:
Typo in package name. Also those classes should probably be nested into something like `com.meemaw.shared.rest.query`